library/base/README.md

```diff
@@ -0,0 +1,9 @@
+# base
+
+This directory contains the definition for the `unikraft.org/base:latest` image.
```

Comment:
Mention the `base` image is a Unikraft runtime built from [`app-elfloader`](https://github.com/unikraft/app-elfloader) that loads and runs Linux native binaries using the [binary-compatibility mode](https://unikraft.org/docs/concepts/compatibility).